connectivity/tests/common.go

```diff
@@ -48,3 +54,81 @@ func hasAllLabels(labelsContainer labelsContainer, filters map[string]string) bo
 	}
 	return true
 }
+
+type retryCondition struct {
+	podLabels map[string]string
+	all       bool
+	destPort  uint32
+	destIP    string
+}
+
+// CurlOptions returns curl retry option or empty slice depending on retry conditions
+// if all value set it means always return retry options
+// for pod-to-cidr all or destIP is checked
+// for pod-to-world all, destPort and destPort + pod label is checked
```

Comment:
Might make sense to move these comments to the respective `With*() RetryOption` funcs below.

Reply:
👍